plugins/dominant-color-images/hooks.php

```diff
@@ -186,3 +186,82 @@ function dominant_color_render_generator(): void {
 	echo '<meta name="generator" content="dominant-color-images ' . esc_attr( DOMINANT_COLOR_IMAGES_VERSION ) . '">' . "\n";
 }
 add_action( 'wp_head', 'dominant_color_render_generator' );
+
+/**
+ * Adds inline CSS for dominant color styling in the WordPress admin area.
+ *
+ * This function registers and enqueues a custom style handle, then adds inline CSS
+ * to apply background color based on the dominant color for attachment previews
+ * in the WordPress admin interface.
+ *
+ * @since n.e.x.t
+ */
+function dominant_color_admin_inline_style(): void {
+	$handle = 'dominant-color-admin-styles';
+	// PHPCS ignore reason: Version not used since this handle is only registered for adding an inline style.
+	// phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
```

Comment:
Nit: The rationale can be added to the same comment per https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#:~:text=If%20required%2C%20you%20can%20add%20a%20note%20explaining%20why%20the%20file%20is%20being%20ignored%20by%20using%20the%20%2D%2D%20separator.

```suggestion
	// phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion -- Version not used since this handle is only registered for adding an inline style.
```